packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div id="oc-file-details-sidebar">
+    <oc-loader v-if="loading" />
+    <div v-else>
+      <div
+        v-if="highlightedFile.preview"
+        class="details-preview uk-flex uk-flex-middle uk-flex-center oc-mb-m"
+      >
+        <oc-img :src="highlightedFile.preview" alt="" />
```

Comment:
can we add the resource name as alt here

Reply:
Thanks for the suggestion! Matter of fact we can't since there is no way to be sure there is (and no way for the user to add) enough context to make this a11y-compliant. An `alt`-tag that is intentionally left completely blank tells screenreaders to ignore the element as "not adding any valuable context" :)